tests/func/test_app.py

```diff
@@ -37,6 +37,37 @@ def silent(body: Body):
         return body
 
 
+@pytest.fixture
+def app_raise_on_validation_error(app):
+    app.config["FLASK_PYDANTIC_VALIDATION_ERROR_RAISE"] = True
+
+    def validation_error(error: ValidationError):
+        return (
+            jsonify(
+                {
+                    "title": "validation error",
+                    "body": error.body_params,
+                }
+            ),
+            422,
+        )
+
+    app.register_error_handler(ValidationError, validation_error)
+
+    class Body(BaseModel):
+        param: str
+
+    @app.route("/no_params", methods=["POST"])
+    @validate()
+    def no_params(body: Body):
```

Comment:
Do we need this route? I don't see it being used anywhere.

Reply:
Quite right. I started with an existing test, copied it, and modified. Left that bit in. Removed now.